### ISSUE TITLE
Can i replace all "@" symbols with "~" ?

### ISSUE THREAD

**Issue by capsload2** [reactiflux/discord-irc#563]
Hey!

Thanks for this awesome project. I love it.

Is there any way i can replace all written " @ " chars into " ~ " from Discord to Twitch and Twitch to Discord?

Thanks!

**Comment by Throne3d** [comment]
Hi there! This project is currently in maintenance mode, so it's unlikely that we'll implement this feature in our own time. However, I've been merging well-crafted pull requests when I see them! If someone would be interested in adding this feature (behind a config and with a test), I could try to get it added to the project.

I'll mark this issue as a feature request + help wanted for now.

**Comment by capsload2** [comment]
Thanks man!

Wouldnt it also work if i modify some of bot.js lines?

**Comment by Throne3d** [comment]
Yep, I think so! Maybe [here](https://github.com/reactiflux/discord-irc/blob/d646ba149fe62b773eb8091d0eadf20dfcd4c83e/lib/bot.js#L283) to use ~ instead of @, to get it to mention Twitch users correctly? Then there's a section of code [down later](https://github.com/reactiflux/discord-irc/blob/d646ba149fe62b773eb8091d0eadf20dfcd4c83e/lib/bot.js#L506) to match mentions in IRC, which you might be able to modify to get it to work right.

**Comment by capsload2** [comment]
Not working. It still mentioned the Guy in the Discord with @nickname even when i replace the line with ~
